pkg/provider-local/controller/controlplane/valuesprovider.go

```diff
@@ -71,7 +71,7 @@ func getSecretConfigs(namespace string) []extensionssecretsmanager.SecretConfigW
 		{
 			Config: &secretsutils.BasicAuthSecretConfig{
 				Name:           local.Name + "-dummy-auth",
-				Format:         secretsutils.BasicAuthFormatCSV,
+				Format:         secretsutils.BasicAuthFormatNormal,
```

Comment:
@rfranzke is it safe to change this? What is the purpose of this secret?

Reply:
Yes, that's just a dummy secret for the e2e credentials rotation test. And it's provider-local anyways, so we should be good :)